oxidation/libparsec/crates/crypto/src/sodiumoxide/sign.rs

```diff
@@ -109,6 +109,20 @@ impl VerifyKey {
     pub fn verify(&self, signed: &[u8]) -> Result<Vec<u8>, CryptoError> {
         verify(signed, &self.0).or(Err(CryptoError::SignatureVerification))
     }
+
+    /// Verify a signature using the given [VerifyKey], `signature` and `message`
```

Comment:
Missing tests for `verify_with_signature` in `tests/sign.rs` if this method went missing unnoticed